react-backend/app.js

```diff
@@ -6,7 +6,9 @@ const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 
 const qrGenerator = require('./routes/qr_generator');
+const listAllUsers = require('./routes/list_all_users');
```

Comment:
I've been trying to group new files by name in a more organised way, so `users_all` and `users_filtered`.

Reply:
And the variable names matched by camel rather than snake (usersAll, etc.)